forest/jasmine/mobmat2traj.py

```diff
@@ -972,6 +972,11 @@ def impute_gps(
     # for observed flights, observed pauses, and missing intervals
     flight_table, pause_table, mis_table = create_tables(mob_mat, bv_subset)
 
+    if len(flight_table) == 0:
+        sys.exit("No flight observed in the data.")
```

Comment:
Since Forest is a library that can be used in a Django app and/or Celery task, it's better to raise an exception instead of using `sys.exit()`